package.json

```diff
@@ -43,8 +43,8 @@
     "compile:current": "cross-env MODE=production npm run build && electron-builder build --config .electron-builder.config.cjs",
     "test": "npm run test:unit && npm run test:e2e",
     "test:unit": "npm run test:main && npm run test:preload && npm run test:preload-docker-extension && npm run test:renderer && npm run test:tools && npm run test:extensions",
-    "test:e2e": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && npm run test:e2e:run",
-    "test:e2e:build": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build",
+    "test:e2e": "npm run test:e2e:build && npm run test:e2e:run",
```

Comment:
As per discussion on PR, please revert running the install part in tests, and put it into a postinstall.

```
...
"prepare": "husky install",
"postinstall": "npx playwright install webkit ffmpeg"
```

Reply:
@benoitf Like this?